Client/Extensions/UIView+Extension.swift

```diff
@@ -99,4 +99,13 @@ extension UIView {
         maskLayer.path = maskPath.cgPath
         layer.mask = maskLayer
     }
+
+    func getAllSubviews<T>(ofType type: T.Type) -> [T] {
+        var secondLevelSubviews = [T]()
+        let firstLevelSubviews: [T] = subviews.compactMap { view in
+            secondLevelSubviews = secondLevelSubviews + view.getAllSubviews(ofType: type)
+            return view as? T
+        }
+        return firstLevelSubviews + secondLevelSubviews
+    }
 }
```

Comment:
This doesn't need to be an extension since it just has this one use case, it can be a private func that takes the view as a parameter that lives inside Themable.